pbhmergers.py

```diff
@@ -42,7 +42,13 @@ def __init__(self,*args,conc_model="ludlow", conc_value=1.,hubble=0.67, **kwargs
 
     def get_nu(self,mass):
         """Get nu, delta_c/sigma"""
+<<<<<<< HEAD
+        if self.ureg.get_dimensionality('') == self.ureg.get_dimensionality(mass):
+            mass = mass * self.ureg.Msolarh
+        return 1.686/self.overden.sigmaof_M_z(mass.to(self.ureg.Msolar).magnitude)
+=======
```

Comment:
You have the right answer but weird version control markers here. My guess is you tried this in a branch and somehow conflicted? The way to fix that is to open the file and delete the version control markers by hand before committing.

Reply:
I took one off for the version control as a whole but did not otherwise penalize you